src/deploy/git.py

```diff
@@ -8,4 +9,11 @@ def log(commit_id):
     """
     Returns a one-line log for a given commit ID.
     """
-    return cmd("git", "show", "-s", "--format=%B", commit_id)
+    try:
+        cmd("git", "fetch", "origin")
+
+        # %s = subject, the first line of the commit
+        # See https://git-scm.com/docs/pretty-formats
+        return cmd("git", "show", "-s", "--format=%s", commit_id)
+    except subprocess.CalledProcessError:
+        return ""
```

Comment:
Do we not want to return any log here if we are catching the exception?

Reply:
No; it's just used for pretty-printing in a table. If it fails you'll see an empty cell in the list of commits, but I don't want to clutter up the output with errors that don't affect the release itself.